src/Bytes/Docraptor/Client.php

```diff
@@ -60,7 +60,7 @@ public function __construct(HttpClientInterface $httpClient, $apiKey)
             ->setBaseUrl('https://docraptor.com')
             ->setDefaultHeaders(array(
                 'Authorization' => 'Basic '.base64_encode($apiKey.':'),
-                'User-Agent' => 'Bytes Docraptor client',
+                'User-Agent' => 'Bytes DocRaptor client',
```

Comment:
This is probably the weirdest change. Don't know if you want this to be updated, but I made it. Also, it would be nice that if/when this is a versioned library, the version was part of the User-Agent.